server/get-server-options.js

```diff
@@ -2,21 +2,21 @@ module.exports = function getServerOptions(options) {
   options = options || {};
 
   return {
-    connections: {
-      routes: {
-        security: {
-          hsts: {
-            maxAge: 15768000,
-            includeSubDomains: true,
-            preload: true
-          },
-          xframe: process.env.ENABLE_XFRAMEOPTIONS === 'true',
-          xss: true,
-          noOpen: true,
-          noSniff: true
-        }
+    address: process.env.HOST,
+    port: process.env.PORT,
+    uri: process.env.APPLICATION_URI,
```

Comment:
These above three are new data. Not that there is another wrong, but is it something required now or is it new data we need access to later in the app?

Reply:
Hapi 17 deprecated the "Connection" API, which we passed this information to in server/index.js. You now set up this stuff in the main server config.